### PR TITLE
mark CSIMigrationAzureFile as GA

### PR DESCRIPTION
**Pull request by andyzhangx**
mark CSIMigrationAzureFile as GA, refer to https://github.com/kubernetes/kubernetes/pull/113160

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -228,6 +225,10 @@ For a reference to old feature gates that are removed, please refer to
 | `CSIMigrationAzureDisk` | `false` | Beta | 1.19 | 1.22 |
 | `CSIMigrationAzureDisk` | `true` | Beta | 1.23 | 1.23 |
 | `CSIMigrationAzureDisk` | `true` | GA | 1.24 | |
+| `CSIMigrationAzureFile` | `false` | Alpha | 1.15 | 1.20 |
+| `CSIMigrationAzureFile` | `false` | Beta | 1.21 | 1.23 |
+| `CSIMigrationAzureFile` | `true` | Beta | 1.24 | |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:230]:
```suggestion
| `CSIMigrationAzureFile` | `true` | Beta | 1.24 | 1.25 |
```

**Review by sftim — COMMENTED** [review]
Thanks.

Please have a look at https://kubernetes.io/docs/concepts/storage/volumes/#azurefile  - there may be some more content to update.

**Comment by andyzhangx** [comment]
> Thanks.
> 
> Please have a look at https://kubernetes.io/docs/concepts/storage/volumes/#azurefile - there may be some more content to update.

@sftim thanks, I have updated.

**Comment by tengqm** [comment]
This one should be proposed against the `dev-1.26` branch rather than the `main` branch.

**Comment by xing-yang** [comment]
The content looks good to me. As mentioned by @tengqm, please submit to dev-1.26 branch.

**Comment by sftim** [comment]
Given https://github.com/kubernetes/website/pull/37813#issuecomment-1308835931

/hold

OK to unhold once that's fixed.

**Comment by andyzhangx** [comment]
opened a new PR for dev-1.26 branch: https://github.com/kubernetes/website/pull/37825

**Comment by jsafrane** [comment]
/close 
in favor of https://github.com/kubernetes/website/pull/37825

**Pull request closed without merging**